### PR TITLE
Issue #25 - adding more meaningful ObjectMetadata

### PR DESCRIPTION
**Pull request by prondzyn**

### CODE REVIEW

**Comment by dmarkov** [comment]
@prondzyn Thanks, let me find someone who can review this pull request

**Comment by dmarkov** [comment]
@caarlos0 it's yours, please review

**Comment by prondzyn** [comment]
@dmarkov I see that Travis CI build failed. I probably found solution of the problem. Maybe I first fix it and prepare second pull request before the @caarlos0 code review?

**Comment by caarlos0** [comment]
@prondzyn yes please, fix the build and then let me know to review it.

Thanks!

File: src/test/java/com/jcabi/s3/mock/MkOcketTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+     * @throws Exception If fails
+     */
+    @Test
+    public void readContentTypeAndLengthFromMetadata() throws Exception {
+        final Region region = new MkRegion(this.temp.newFolder());
+        final Bucket bucket = region.bucket("test");
+        final Ocket ocket = bucket.ocket("hello.txt");
+        final String text = "hello, world!";
+        new Ocket.Text(ocket).write(text);
+        final Ocket readOcket = new Ocket.Text(bucket.ocket(ocket.key()));
```
Comment (pinaf) [inline src/test/java/com/jcabi/s3/mock/MkOcketTest.java]: @prondzyn let's rename this ocket to `read` and the other ocket to `write`.

File: src/test/java/com/jcabi/s3/mock/MkOcketTest.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        final ObjectMetadata metadata = readOcket.meta();
+        Assert.assertEquals("text/plain", metadata.getContentType());
+        Assert.assertEquals(text.length(), metadata.getContentLength());
```
Comment (pinaf) [inline src/test/java/com/jcabi/s3/mock/MkOcketTest.java]: @prondzyn no test for the new `Headers.DATE` key ?

**Comment by dmarkov** [comment]
@caarlos0 I just added **16 mins** to your account, many thanks for your contribution (AP-8X587510EW390425P).. 4 hours and 24 mins spent here; review comments (c=1) added as a bonus; +16 added to your rating, current score is: [+989](http://www.netbout.com/b/35095?open=rating)

**Pull request closed without merging**